### ISSUE TITLE
Object metadata in HIL

### ISSUE THREAD

**Issue by naved001** [CCI-MOC/hil#810]
It would be nice to have time stamps like "date created" or "last modified" associated with objects like Projects and/or nodes. This would make a HIL GUI have some useful information for the end user.

This was suggested by @xuhang57

**Comment by zenhack** [comment]
We already have something like this for nodes (originally added for some
use case to do with TPMs IIRC, but it's a generic mechanism). I could
see extending it to most/all objects.

Is that the kind of thing you had in mind?

Quoting Naved Ansari (2017-06-27 14:10:07)

**Comment by naved001** [comment]
> Is that the kind of thing you had in mind?

Yeah, that, or if there is a way to know when the rows are created/modified in a table.

**Comment by zenhack** [comment]
It's not obvious to me how we would do that short of actually updating a field every time we do a modification, which I'm not a fan of, though maybe there's a way. But I also want to err on the side of keeping complexity out of HIL core, and I don't know that I think it's a good trade-off from that perspective.
